.github/workflows/maven.yml

```diff
@@ -0,0 +1,32 @@
+name: Java CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+      - dev  
+
+jobs:
+  lint:
+    name: Static Code Analysis With Ktlint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
This was the error

Reply:
Thats what happens when you copy and paste. 
![](https://media.giphy.com/media/CLmanYaDhshos/giphy.gif)